brainglobe_stitch/stitching_widget.py

```diff
@@ -150,6 +164,37 @@ def __init__(self, napari_viewer: Viewer):
         self.add_tiles_button.setEnabled(False)
         self.layout().addWidget(self.add_tiles_button)
 
+        self.select_imagej_path = QWidget()
+        self.select_imagej_path.setLayout(QHBoxLayout())
+
+        self.imagej_path_text_field = QLineEdit()
+        self.imagej_path_text_field.setText(str(self.default_directory))
+        self.imagej_path_text_field.editingFinished.connect(
```

Comment:
The 4 masking block are the other channel which is dimmer than the first channel. I've not set it to adjust the contrast per channel when loading to avoid this issue.